### DISCUSSION TITLE
dynamic tenantDomain

### DISCUSSION THREAD

**Discussion by khalidmaquilang** [filamentphp/filament#13697] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

How can I make this dynamic? There will be tenants that will only use subdomain and the others will use their own domain.. how can I achieve that using `tenantDomain` function

**Comment by bilogic** [comment]
Yea, that and how do we let 1 tenant use both `{tenant:slug}.my-filament-app-domain.com` and `{tenant:domain}` at the same time?
